app/src/main/java/com/simplekjl/letschat/MainActivity.kt

```diff
@@ -19,13 +34,23 @@ class MainActivity : AppCompatActivity() {
 
     val ANONYMOUS = "anonymous"
     val DEFAULT_MSG_LENGTH_LIMIT = 1000
+    val RC_SIGN_IN: Int = 12
```

Comment:
These can be private if not being used in other classes.

Reply:
I fixed them on the next branch, adding them into the companion object :D